test/test_jit.py

```diff
@@ -964,16 +969,7 @@ def fn(x, y):
 
         graph = torch.jit.script(fn).graph
         self.run_pass('cse', graph)
-        self.assertExpectedGraph(graph)
-
-    def test_scalar(self):
```

Comment:
I think we still need it to check that if we don't provide any type annotations, the scalar to tensor promotion are still triggered correctly.

Reply:
there is no scalar to tensor promotion, and that the default types is tensors is tested elsewhere